lib/src/command/login.dart

```diff
@@ -58,8 +56,8 @@ class LoginCommand extends PubCommand {
 }
 
 class _UserInfo {
-  final String name;
-  final String email;
+  final String? name;
+  final String? email;
```

Comment:
I think the email should be non-nullable, otherwise the `toString` is broken as-is